### ISSUE TITLE
Text becomes unreadable

### ISSUE THREAD

**Issue by mnsnownutt** [GTNewHorizons/Angelica#763]
### Your GTNH Discord Username

mnsnownutt

### Your Pack Version

2.6.1

### Your Server

Local

### Java Version

Java 21

### Type of Server

Single Player

### Your Expectation

It varies from anywhere from 1-15 minutes, but then all of the sudden all text becomes unreadable as pictured below.  I asked on the Discord and no one responded to me at all, tried posting on Reddit and no one had any suggestions to fix the issue.  I have a little over 1,000 hours on this world and this just started happening.

![image](https://github.com/user-attachments/assets/7ab56ffc-296b-4a09-bb0a-5157f0b9b5e0)

### The Reality

NA

### Your Proposal

That it would not happen.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [x] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by mnsnownutt** [comment]
Note, I updated to 2.7.2 and it is still happening, but not nearly as frequently as when I was on 2.6.1.

**Comment by YamiKami-Sama** [comment]
Got the same in 2.7.2 and 2.7.3 (with java 17.0.9), but now it happend every 5min, it begin to be unplayable.
Some tips for those ho got this rarely, you can just do a F3 + T to reload the assets and it fix it, i don't know if reloading the game make it appear less frequantly or not.

**Comment by GribanovIvan** [comment]
pro tip: disable Angelica's font rendering till fix

**Comment by mnsnownutt** [comment]
If you are talking about this, it does not fix the issue.  Is there another one?

    # Batch drawScreen fonts [Experimental] [default: true]
    B:enableFontRenderer=false

**Comment by YamiKami-Sama** [comment]
For me it seem that the issue (at least when it happen every 1-5min) was comming from having quad-coprocessing (2 or 3) in "Place Item in World", i also have a thaumcraft wand in place item in world, but he doesn't seem to cause issue.

**Comment by MalTeeez** [comment]
Since Font Rendering was heavily reworked, it is unlikely that this issue still occurs on the latest version. It it does for anyone, please comment below.

**Comment by mnsnownutt** [comment]
Yes, I am happy to report that since updating to 2.8.4 it has no longer occurred.
